try-it.md

```diff
@@ -479,37 +472,25 @@ image built from the [metal3-io/baremetal-operator](https://github.com/metal3-io
 First, you must scale down the deployment of the `baremetal-operator` running
 in the cluster.
 
-```sh
-kubectl scale deployment capm3-baremetal-operator-controller-manager -n metal3 --replicas=0
+```console
+$ kubectl scale deployment baremetal-operator-controller-manager -n baremetal-operator-system --replicas=0
 ```
 
 To be able to run `baremetal-operator` locally, you need to install
 [operator-sdk](https://github.com/operator-framework). After that, you can run
 the `baremetal-operator` including any custom changes.
 
-```sh
-cd ~/go/src/github.com/metal3-io/baremetal-operator
-make run
+```console
+$ cd ~/go/src/github.com/metal3-io/baremetal-operator
+$ make run
 ```
 
 ### 2.5. Running Custom Cluster API Provider Metal3
 
 There are two Cluster API related managers running in the cluster. One
 includes set of generic controllers, and the other includes a custom Machine
-controller for Metal3. If you want to try changes to
-`cluster-api-provider-metal3`, you want to shut down the custom Machine
-controller manager first.
-
-```sh
-$ kubectl scale deployment capm3-controller-manager -n capm3-system --replicas=0
-```
-
-Then you can run the custom Machine controller manager out of your local git tree.
-
-```sh
-cd ~/go/src/github.com/metal3-io/cluster-api-provider-metal3
-make run
-```
+controller for Metal3. You can try changes to
+`cluster-api-provider-metal3` and the changes will be automaticly applied when using Tilt.
```

Comment:
we haven't talked about Tilt until this point and not sure if the mention of it will not confuse people.
I think either we need to explain how to work with Tilt/give a reference to a document where we have explained or
keep this part as it is